### ISSUE TITLE
composer@2.0.11: download via aria2 failed

### ISSUE THREAD

**Issue by NatoBoram** [ScoopInstaller/Main#1907]
```yaml
Updating 'composer' (2.0.8 -> 2.0.11)
Downloading new version
Starting download with aria2 ...
Download: Exception: [AbstractCommand.cc:313] errorCode=1 Network problem has occurred. cause:A socket operation was attDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: f78f51|ERR |       0B/s|C:/Users/natob/Scoop/cache/composer#2.0.11#https_getcomposer.org_download_2.0.11_composer.phar
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 1) An unknown error occurred
ERROR https://getcomposer.org/download/2.0.11/composer.phar
    referer=https://getcomposer.org/download/2.0.11/
    dir=C:\Users\natob\Scoop\cache
    out=composer#2.0.11#https_getcomposer.org_download_2.0.11_composer.phar

ERROR & 'C:\ProgramData\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\natob\Scoop\cache\composer.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.2 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=16 --max-connection-per-server=16 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=12184 --continue --summary-interval 0

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=composer%402.0.11%3a+download+via+aria2+failed
```

**Comment by NatoBoram** [comment]
That doesn't seem right 🤔 
```yaml
composer: 2.0.8 -> 2.0.11 (global)
Updating one outdated app:
Updating 'composer' (2.0.8 -> 2.0.11)
Downloading new version
composer.phar (2.1 MB) [======================================================================================] 100%
Checking hash of composer.phar ... ERROR Hash check failed!
App:         main/composer
URL:         https://getcomposer.org/download/2.0.11/composer.phar
First bytes: 23 21 2F 75 73 72 2F 62
Expected:    eabf2917072096a94679193762501319e621e2b369a4a1256b2c27f4e6984477
Actual:      d6eee0d4637f4bd82bdae098fceda300dcb3ec35bf502604fbe7510933b8f952

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=composer%402.0.11%3a+hash+check+failed
```

**Comment by chawyehsu** [comment]
fixed by #1909
